docs/helm/helm_get_version.md

```diff
@@ -0,0 +1,42 @@
+## helm get version
```

Comment:
This file is not needed and should be removed. All docs in docs/helm are generated file from the text specified in the helm command definition.  Ref: https://helm.sh/docs/developers/#man-pages